libs/sdk-core/src/breez_services.rs

```diff
@@ -1306,27 +1305,20 @@ impl Receiver for PaymentReceiver {
     }
 }
 
-fn validate_opening_fee_params_menu(
-    opening_fee_params_menu: &Vec<models::OpeningFeeParams>,
-) -> Result<()> {
+fn validate_opening_fee_params_menu(opening_fee_params_menu: &Vec<OpeningFeeParams>) -> Result<()> {
```

Comment:
NIT: rename to `validate_opening_fee_params_options` ?

Reply:
It's called menu in the spec, probably best to stick with the spec naming here.

Reply:
I moved this to `OpeningFeeParamsMenu::validate()`.